pilot/pkg/networking/core/v1alpha3/listener_builder.go

```diff
@@ -386,9 +386,7 @@ func (lb *ListenerBuilder) buildHTTPConnectionManager(httpOpts *httpListenerOpts
 		filters = append(filters, xdsfilters.GrpcWeb)
 	}
 
-	if httpOpts.protocol.IsGRPC() {
-		filters = append(filters, xdsfilters.GrpcStats)
-	}
+	filters = append(filters, xdsfilters.GrpcStats)
```

Comment:
If we always enable grpc stats, why nor grpcweb as well?

Reply:
Sure we can do. But let us do it separately.